drake/bindings/python/pydrake/test/testRBTCoM.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function
```

Comment:
just for the record -- pretty sure we've resolved those path issues (it wasn't due to the python 2 vs 3, but the path)